light_curves/light_curve_generator.md

```diff
@@ -362,7 +386,7 @@ n_workers = 8
 heasarc_kwargs = dict(catalog_error_radii={"FERMIGTRIG": "1.0", "SAXGRBMGRB": "3.0"})
 ztf_kwargs = dict(nworkers=None)  # the internal parallelization is incompatible with Pool
 wise_kwargs = dict(radius=1.0, bandlist=['W1', 'W2'])
-panstarrs_kwargs = dict(radius=1.0/3600.0)
+panstarrs_search_radius = 1.0 # arcsec
```

Comment:
nitpick, but I think it would ultimately be nice if these radiuses could be any astropy quantity rather than a float, of which the user should keep track if arcsec, or arcmin, or degrees, depending on which datasets they are querying. This comment likely will go beyond this PR into a follow-up

Reply:
I turned this into a new issue to track it, but not necessarily add it to this PR.

Reply:
> it would ultimately be nice if these radiuses could be any astropy quantity

Generally, I agree, but in this case (as I recall) the scale_up.md notebook needs these to be floats (or ints) because the functions are called from the command line. So if they get changed to astropy quantities we will need to implement a workaround in scale_up.md.

OTOH, we'll probably want to revisit the parallel cell in this notebook because this PR removes Pan-STARRS from the `multiprocessing` call so now it has to wait for the others to finish and then it runs separately. It would be nice to get them all to actually run in parallel again. Potentially, we can come up with a solution that actually works at scale, and then we wouldn't need scale_up.md anymore. The two biggest reasons we wrote scale_up.md[*] in the first place were that (1) the notebook would not continue running long enough for the parallel cell to complete at scale (servers were being shut down automatically) and (2) the ZTF function was taking the longest by far so we implemented `multiprocessing` directly in that function, but that could not be used in conjunction with the `multiprocessing` in the notebook's parallel cell. The first might be solved by infrastructure updates and the second by getting ZTF into HATS format -- if so, maybe we can switch `multiprocessing` -> `dask` and run all the calls in parallel with Dask directly from this notebook, at scale.

[*] scale_up.md describes the underlying bash code but doesn't actually execute it at scale -- the user is instructed to do that directly from the command line.